requirements.txt

```diff
@@ -3,7 +3,7 @@ astropy >= 4.0
 esutil >= 0.6.9
 galsim >= 2.4.6
 git+https://github.com/lsst/daf_butler@main#egg=lsst-daf-butler
```

Comment:
Whilst you are in here, can you modernize the format of the file?

```
lsst-daf-butler @ git+https://github.com/lsst/daf_butler@main
lsst-pipe-base @ git+https://github.com/lsst/pipe_base@main
lsst-utils @ git+https://github.com/lsst/utils@main
lsst-resources @ git+https://github.com/lsst/resources@main
lsst-pex-config @ git+https://github.com/lsst/pex_config@main
```

Reply:
Oops, I just realized I confused this message with the one in ctrl_mpexec (which I did) and didn't do it.  @leeskelvin, could you do this next time you're in the area?

Reply:
Actually it looks like I'll be doing it myself on a hotfix branch now because I also failed to drop the branch dependencies.  Really botched this merge.

Reply:
Consider copying over the middleware checker for exactly that situation.

Reply:
Fixed on #36, but I didn't include the middleware checker.